src/Persistence.Memory/InMemoryAggregateRepository.cs

```diff
@@ -1,21 +1,39 @@
-﻿using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Linq;
-using CR.AggregateRepository.Core;
-using CR.AggregateRepository.Core.Exceptions;
+﻿// <copyright file="InMemoryAggregateRepository.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
 
 namespace CR.AggregateRepository.Persistence.Memory
 {
+    using System;
+    using System.Collections.Concurrent;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Core;
+    using Core.Exceptions;
+
+    /// <summary>
+    /// In Memory implementation of the Aggregate Repository.
+    /// </summary>
     public class InMemoryAggregateRepository : IAggregateRepository
     {
-        public readonly ConcurrentDictionary<object,List<object>> EventStore = new ConcurrentDictionary<object, List<object>>();
-        
+        /// <summary>
+        /// In memory event storage.
```

Comment:
Inconsistent capitalization of memory between this & the class summary. In-memory should probably also be hyphenated.